### ISSUE TITLE
Use t distribution for ci.force confidence intervals

### ISSUE THREAD

**Issue by leifeld** [leifeld/texreg#7]
Comment from user [yennywebbv](https://r-forge.r-project.org/users/yennywebbv/) (Yenny Webb Vargas) on R-Forge:

Hi!

Thanks for the great implementation!

I have an important suggestion: add an option to use a t distribution, with appropriate degrees of freedom taken from the model. As it is implemented, the confidence intervals do not match the ``lm`` ``confint`` ones. This is due to ``texreg`` using a standard normal distribution, as opposed to a t distribution that pulls the error degrees of freedom from the model.

I live in a small-sample world, and the degrees of freedom really make a big difference.

I attach code for a possible implementation of this. You would need to update the ``getdata`` function to output the degrees of freedom of the model, and have access to it in the S4 object (``models[[i]]@df.residual``).

Thanks!

-Y

```
ciforce = function (models, ci.force = rep(FALSE, length(models)), ci.level = 0.95,tdist=FALSE) 
{
  if (class(ci.force) == "logical" && length(ci.force) == 1) {
    ci.force <- rep(ci.force, length(models))
  }
  if (class(ci.force) != "logical") {
    stop("The 'ci.force' argument must be a vector of logical values.")
  }
  if (length(ci.force) != length(models)) {
    stop(paste("There are", length(models), "models and", 
               length(ci.force), "ci.force values."))
  }
  for (i in 1:length(models)) {
    if (ci.force[i] == TRUE && length(models[[i]]@se) > 0) {
      if(tdist==TRUE){
        quant <- qt(1 - ((1 - ci.level)/2), models[[i]]@df.residual)
      }else{
        quant <- qnorm(1 - ((1 - ci.level)/2))
      }
      upper <- models[[i]]@coef + (quant * models[[i]]@se)
      lower <- models[[i]]@coef - (quant * models[[i]]@se)
      models[[i]]@ci.low <- lower
      models[[i]]@ci.up <- upper
      models[[i]]@se <- numeric(0)
      models[[i]]@pvalues <- numeric(0)
    }
  }
  return(models)
}
```

**Comment by lingchuding** [comment]
@leifeld Hi, I am a novice of R. These days I am using texreg to export results from analyses. And I hope to obtain the p values of t-statistics automatically when using texreg. It seems that someone has worked out the solution. But I do not understand the code above very clearly. Could you share with me how to realize it? Hope to get suggestions from you. Thanks!

**Comment by czucca** [comment]
closed by PR leifeld#98
